lmnet/tests/lmnet_tests/datasets_tests/test_mscoco.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import pytest
+
+from lmnet.datasets.mscoco import MscocoObjectDetection, DEFAULT_CLASSES
+
+
+pytestmark = pytest.mark.usefixtures("set_test_environment")
+
+
+def test_mscoco_object_detection_optional_classes_instantiate():
+    classes = [DEFAULT_CLASSES[0]]
+    MscocoObjectDetection.classes = classes
+    try:
+        MscocoObjectDetection()
+        assert True
+    except Exception:
+        assert False
```

Comment:
I think it's better to assert more clear expectations in this test case like setting the number of classes the `MscocoObjectDetection()` method returns. not just by setting a try-catch that causes unmet expectations.

Reply:
Also, I can't recommend using try-catch in unit tests.

Reply:
Maybe you can refer to some test cases here:

https://github.com/blue-oil/blueoil/blob/master/lmnet/tests/lmnet_tests/datasets_tests/test_cityscapes.py

Reply:
@Joeper214 I add another test. please check this out 🐶